src/webaudio-mediasource.js

```diff
@@ -0,0 +1,459 @@
+import WebAudio from './webaudio';
+import * as util from './util';
+
+/**
+ * MediaElement backend
+ */
+export default class WebAudioMedia extends WebAudio {
+    /** @private */
+    static scriptBufferSize = 256;
+    /** @private */
+    audioContext = null;
+    /** @private */
+    offlineAudioContext = null;
+    /**
+     * Construct the backend
+     *
+     * @param {WavesurferParams} params Wavesurfer parameters
+     */
+    constructor(params) {
+        super(params);
+        /** @private */
+        this.params = params;
+        /** @private */
+        this.params = params;
```

Comment:
duplicated code

Reply:
How does this work? Can I modify code deleting duplicated code or in this phase do you make optimizations?

Reply:
You can commit changes in your local repo and push to your GitHub fork's master.